### PR TITLE
change plugin tutorials for humble

### PR DESCRIPTION
**Pull request by Ryujiyasu**
Fix points

1. change "Recovery" to "Behavior"
2. delete  Eloquent and earlier info (because EOL version)

### CODE REVIEW

File: plugin_tutorials/docs/writing_new_behavior_plugin.rst [hunk 1]
```diff
@@ -178,27 +178,86 @@ It is good practice to place these lines at the end of the file but technically,
...
     <build_type>ament_cmake</build_type>
-    <nav2_core plugin="${prefix}/recovery_plugin.xml" />
+    <nav2_core plugin="${prefix}/behavior_plugin.xml" />
   </export>
 
 5. Compile and it should be registered. Next, we'll use this plugin.
 
-3- Pass the plugin name through params file
--------------------------------------------
+3- Pass the plugin name through params file (Galactic and earlier)
```
Comment (SteveMacenski) [inline plugin_tutorials/docs/writing_new_behavior_plugin.rst:194]: Humble and newer should be first, then the second copy of Galactic and older should just specify the changes to the Humble and newer code block (e.g. change `behavior_server` to `recovery_server` in namespace, but that's it?). We don't need to make this excessively long

**Review by SteveMacenski — CHANGES_REQUESTED** [review]

File: plugin_tutorials/docs/writing_new_behavior_plugin.rst (hunk 1 above)
Comment (Ryujiyasu) [reply plugin_tutorials/docs/writing_new_behavior_plugin.rst:194]: ok

Comment (Ryujiyasu) [reply plugin_tutorials/docs/writing_new_behavior_plugin.rst]: - [ ] Humble and newer ->first
- [ ] shorten sentence

thank you!!

File: plugin_tutorials/docs/writing_new_behavior_plugin.rst [hunk 2]
```diff
@@ -178,32 +178,35 @@ It is good practice to place these lines at the end of the file but technically,
...
-    <nav2_core plugin="${prefix}/recovery_plugin.xml" />
+    <nav2_core plugin="${prefix}/behavior_plugin.xml" />
   </export>
 
 5. Compile and it should be registered. Next, we'll use this plugin.
 
+
 3- Pass the plugin name through params file
--------------------------------------------
+--------------------------------------------------------------
```
Comment (SteveMacenski) [inline plugin_tutorials/docs/writing_new_behavior_plugin.rst:196]: This line needs to be reset to the same length as the message above it (or the compiler complains)

File: plugin_tutorials/docs/writing_new_behavior_plugin.rst [hunk 3]
```diff
@@ -178,32 +178,35 @@ It is good practice to place these lines at the end of the file but technically,
...
 
-To enable the plugin, we need to modify the ``nav2_params.yaml`` file as below to replace following params
+To enable the plugin, we need to modify the ``nav2_params.yaml`` file as below to replace following params_file
```
Comment (SteveMacenski) [inline plugin_tutorials/docs/writing_new_behavior_plugin.rst:198]:
```suggestion
To enable the plugin, we need to modify the ``nav2_params.yaml`` file as below to replace following params
```

**Review by SteveMacenski — COMMENTED** [review]
LGTM, just 2 small changes

File: plugin_tutorials/docs/writing_new_behavior_plugin.rst (hunk 2 above)
Comment (Ryujiyasu) [reply plugin_tutorials/docs/writing_new_behavior_plugin.rst:196]: OK

File: plugin_tutorials/docs/writing_new_behavior_plugin.rst (hunk 3 above)
Comment (Ryujiyasu) [reply plugin_tutorials/docs/writing_new_behavior_plugin.rst]: ok

**Review by SteveMacenski — APPROVED** [review]

**Comment by SteveMacenski** [comment]
Thanks!

**Pull request merged**